test/language/statements/class/subclass/class-definition-null-proto-missing-return-override.js

```diff
@@ -18,24 +15,25 @@ info: |
      [...]
      b. Let superclass be the result of evaluating ClassHeritage.
   [...]
-  15. If ClassHeritageopt is present and superclass is not null, then set F's
-      [[ConstructorKind]] internal slot to "derived".
+  15. If ClassHeritageopt is present, then set F's [[ConstructorKind]] internal slot to "derived".
   [...]
 
   9.2.2 [[Construct]]
 
   [...]
-  5. If kind is "base", then
-     a. Let thisArgument be ? OrdinaryCreateFromConstructor(newTarget,
-        "%ObjectPrototype%").
-  [...]
   15. Return ? envRec.GetThisBinding().
+
+  8.1.1.3.4 GetThisBinding ( )
+  [...]
+  3. If envRec.[[ThisBindingStatus]] is "uninitialized", throw a ReferenceError exception.
+  [...]
 ---*/
 
 class Foo extends null {
-  constructor() {}
+  constructor() {
+  }
```

Comment:
in a follow up PR, we could verify this is called. the `reachable += 1` approach would be good enough